.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -49,6 +49,13 @@ body:
     validations:
       required: true
 
+  - type: checkboxes
```

Comment:
I had this as a drop down originally, but I think a checkbox makes more sense as the dropdown adds a `'None'` option. Just giving you a tag @sarahetter since you've worked with the YAML format for an issue template before.

Reply:
Yeah, checkbox makes sense to me